src/libsync/account.cpp

```diff
@@ -474,6 +498,13 @@ void Account::setNonShib(bool nonShib)
     } 
 }
 
+void Account::setBundleRequestsIfCapable(bool bundleRequests){
```

Comment:
I don't think we need this.  We should just use what the capability tells.
For debugging purposes, we might want to have an environment variable.